c/driver_manager/adbc_driver_manager.cc

```diff
@@ -425,6 +438,8 @@ SearchPaths GetEnvPaths(const char_type* env_var) {
   std::wstring path_var;
   path_var.resize(required_size);
   _wgetenv_s(&required_size, path_var.data(), required_size, env_var);
+  // Remove null terminator
+  path_var.resize(required_size - 1);
```

Comment:
can we expand this comment to explain why we need to remove the null terminator?